stake-pool/js/src/utils/program-address.ts

```diff
@@ -28,9 +28,14 @@ export async function findStakeProgramAddress(
   programId: PublicKey,
   voteAccountAddress: PublicKey,
   stakePoolAddress: PublicKey,
+  seed?: BN,
 ) {
   const [publicKey] = await PublicKey.findProgramAddress(
-    [voteAccountAddress.toBuffer(), stakePoolAddress.toBuffer()],
+    [
+      voteAccountAddress.toBuffer(),
+      stakePoolAddress.toBuffer(),
+      seed ? seed.toArrayLike(Buffer, 'le', 4) : Buffer.alloc(0),
```

Comment:
nit-up-to-you: I think a `u32` encoder from `buffer-layout` would work here, too.

Reply:
I chose to keep this just because it matches the format of the functions below it. But let me know if that's a big deal. Was trying to make minimal amount of changes to this file just due to the sensitivity of it (and i'm a first time contriubtor :) )